src/utils/getEvents.js

```diff
@@ -16,3 +18,17 @@ export function getEventsOfCity(city, events) {
       })
     : events;
 }
+
+export function getEventsInRange(userCoords, range, events) {
+  return events.filter((event) => {
+    return (
+      getDistance(
+        { latitude: userCoords[0], longitude: userCoords[1] },
+        {
+          latitude: event.location.coordinates[0],
+          longitude: event.location.coordinates[1],
+        }
+      ) <= range
```

Comment:
maybe it is because i dont know the libary, but what is the backwards arrow for here?

Reply:
It's the lesser than operator. It's looks confusing because of the line break.

Reply:
The library has useful formulas e.g. for calculating the distance between two coordinates.